python/servo/testing_commands.py

```diff
@@ -82,6 +82,8 @@ def create_parser_wpt():
                         help="Servo's JSON logger of unexpected results")
     parser.add_argument('--always-succeed', default=False, action="store_true",
                         help="Always yield exit code of zero")
+    parser.add_argument('--no-default-test-types', default=False, action="store_true",
+                        help="Run the default set of test types provided by wptrunner"),
```

Comment:
The name and description of this flag seem contradictory

Reply:
Default in this case refers to the test of test types that are selected by wptrunner when this flag is not present. Is there a better way to describe it?

Reply:
It's true - the default in the command name refers to the user experience and the flags selected by test-wpt. The default in the description refers to the flags selected by wptrunner without any other input.

Reply:
> Default in this case refers to […] when this flag is not present.

In that case, the flag’s description "run the default set" is wrong since setting the flag makes a different set be used than the one called default.

Unless the "by wptrunner" (as opposed to mach) means that it intentionally refers to a different set. In this case, it’s not great to use the same word "default" to refer to two different sets in the flag’s name and its description.